### ISSUE TITLE
Local MOTDs

### ISSUE THREAD

**Issue by MinoMino** [MinoMino/minqlx#2]
As of right now, MOTDs are global. It's shared across all servers with the same DB. Here's some discussion about it from IRC. Feel free to leave your opinion:

```
<@Mino> Actually, I'm not sure anymore. The issue is that with the current MOTD system, I'm not sure how I'd go about determining which server is which.
<@Mino> I guess I could use net_port
<@Mino> But I don't know
<@Mino> So the idea was to use g_motd and let people set that instead, like in Q3.
<@Mino> and just not use the database
<@Mino> or maybe use the database as an motd to fall back to or something if g_motd isn't set
<mattiZed> hm, i guess just setting motd via g_motd would be best way
<mattiZed> do they get changed that much?
<mattiZed> i literally never changed mine, once i set it...
<mattiZed> just telling people to behave and get the fuck off if they dont :D
<@Mino> Yeah
<@Mino> I think I'll just open an issue on Github for now and maybe have people voice their opinion if they want to while I think about it a bit
```

**Comment by MinoMino** [comment]
Also, using `fs_homepath` would probably be a good way to identify server instances. :penguin:

**Comment by mattiZed** [comment]
I would be okay with setting MOTD via cvar g_motd and not dynamically via !-commands but thats just me. If it is feasible to set different MOTD's for each server using ONE database, I think that would be appreciated.

**Comment by MinoMino** [comment]
Addressed in MinoMino/minqlx-plugins@c7d4e541688313e09bcb3c805cd0e37aeb2bcfcf.
